spec/vcr_cassettes/Inventory/Helpers/create_url.yml

```diff
@@ -44,5 +44,5 @@ http_interactions:
       encoding: UTF-8
       string: ''
     http_version: 
-  recorded_at: Thu, 05 May 2016 11:18:36 GMT
```

Comment:
I've removed also the `spec/vcr_cassettes/Inventory/Helpers/get_feeds.yml` cassette from the commit